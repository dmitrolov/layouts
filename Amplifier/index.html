<!doctype html>
<html>
	<head>
		<meta charset="UTF-8">
		<title></title>
		<link rel="stylesheet" href="style6h.css">
		<!-- include google font -->
		<link href='https://fonts.googleapis.com/css?family=Lobster&subset=latin,cyrillic' rel='stylesheet' type='text/css'>
		<link href='https://fonts.googleapis.com/css?family=Droid+Sans' rel='stylesheet' type='text/css'>
		<link href='https://fonts.googleapis.com/css?family=Istok+Web' rel='stylesheet' type='text/css'>
		<link href='https://fonts.googleapis.com/css?family=Anton' rel='stylesheet' type='text/css'>
		<link href='https://fonts.googleapis.com/css?family=Lato:300' rel='stylesheet' type='text/css'>
	</head>
	<body>
		<div class="wraper">
			<div class="line1"></div>
			<div class="header">
				<div class="inner holder">
					<div class="logo left">
						<h1><a href="#">Amplifier</a></h1>
						<div>Reailty Leaves a lot to the imagination</div>
					</div>
					<div class="menu right">
						<ul>
								<li><a href="#">Home</a></li>
								<li><a href="#">Features +</a></li>
								<li><a href="#">Portfolio +</a></li>
								<li><a href="#">Blog</a></li>
								<li><a href="#">Contact</a></li>
						</ul>
						<div class="tel">
							<img src="images/phone.png" alt="#">
							<a href="tel:8005551234">800-555-1234</a>
						</div>
					</div>
				</div>
			</div>
			<div class="slider inner">
				<div class="row1 holder">
					<div class="mainpart left">
						<h1>this is a big title for the slide</h1>
						<p>Lorem augue eros ac pid! Integer etiam nec porta magnis pulvinar etiam nec cum risus ac magna odio turpis, 
						lectus ut, natoque pulvinar, aliquam, pellentesque mattis platea mid integer nunc dignissim nunc?
						</p>
						<p>Placerat tempor scelerisque cum, urna egestas integer. Lorem augue eros ac pid! 
						Integer etiam nec porta magnis pulvinar etiam nec cum risus ac magna odio turpis, lectus ut, 
						natoque pulvinar, aliquam, pellentesque mattis platea mid integer nunc dignissim, nunc? Placerat tempor
						</p>
						<input class="input1" type="button" value="learn more">
						<input class="input2" type="button" value="buy now!">
					</div>
					<div class="right"><img src="http://placehold.it/520x258&text=slider" alt="slide"></div>
				</div>	
				<div class="row2 holder">
					<a href="#"></a>
					<a href="#"></a>
					<a href="#"></a>
					<a href="#"></a>
				</div>			
			</div>
			<div class="action">
				<div class="inner holder">
					<h2 class="left">Lorem Mattis Dolar Sit Turpis Ut Magnis Augue!</h2>
					<input class="input3 right" type="button" value="Continue Reading">
				</div>
			</div>
			<div class="services inner">
				<div class="rubric1 holder">
					<h2 class="left">Services We Provide</h2>
					<div class="line2 right"></div>
				</div>
				<div class="servcon holder">
					<div class="service">
						<h3>Our First Service</h3>
						<div class="holder">
							<p class="desc">
							<img src="images/service1.png" alt="service1">
							Stique quis, turpis, nisi velit ultricies i rna a, tincidunt. 
							Lorem sit cum? Qui aliquamet magna pulvinar est, et ac! 
							Seagittis penatibus nisi velit purus adipiscing cursus odio! 
							Magnis. Porta, sit nec integer et rhoncus et, cras lacus ridiclus.
							</p>
							<p class="read">read more ...</p>
						</div>
					</div>
					<div class="service">
						<h3>Our Second Service</h3>
						<div class="holder">
							<p class="desc">
							<img src="images/service2.png" alt="service2">
							Stique quis, turpis, nisi velit ultricies i rna a, tincidunt. 
							Lorem sit cum? Qui aliquamet magna pulvinar est, et ac! 
							Seagittis penatibus nisi velit purus adipiscing cursus odio! 
							Magnis. Porta, sit nec integer et rhoncus et, cras lacus ridiclus.
							</p>
							<p class="read">read more ...</p>
						</div>
					</div>
					<div class="service">
						<h3>Our Third Service</h3>
						<div class="holder">
							<p class="desc">
							<img src="images/service3.png" alt="service3">
							Stique quis, turpis, nisi velit ultricies i rna a, tincidunt. 
							Lorem sit cum? Qui aliquamet magna pulvinar est, et ac! 
							Seagittis penatibus nisi velit purus adipiscing cursus odio! 
							Magnis. Porta, sit nec integer et rhoncus et, cras lacus ridiclus.
							</p>
							<p class="read">read more ...</p>
						</div>
					</div>
					<div class="service">
						<h3>Our Fourth Service</h3>
						<div class="holder">
							<p class="desc">
							<img src="images/service4.png" alt="service4">
							Stique quis, turpis, nisi velit ultricies i rna a, tincidunt. 
							Lorem sit cum? Qui aliquamet magna pulvinar est, et ac! 
							Seagittis penatibus nisi velit purus adipiscing cursus odio! 
							Magnis. Porta, sit nec integer et rhoncus et, cras lacus ridiclus.
							</p>
							<p class="read">read more ...</p>
						</div>
					</div>
				</div>
			</div>
			<div class="blog inner holder">
				<div class="rubric1 holder">
					<h2 class="left">Latest from the blog</h2>
					<div class="line2 right"></div>
				</div>
				<div class="blogcon holder">
					<div class="post left">
						<h3>Dapibus Integer Ac Nunc Pluvinar Aliquam</h3>
						<p>Posted: April 3rd, 2011 by Jami Gibbs - 2 Comments</p>
						<div class="postcon holder">
							<div class="left"><img src="http://placehold.it/139x139&text=blog" alt="blog"></div>
							<p>Pot ipsum dolor break yo neck, yall amet, phat adipiscing for sure. 
							Owned sapizzle velizzle, for sure volutpizzle, suscipit , gravida vel, arcu. 
							Pellentesque its fo rizzle boofron. Sizzle eros. Yippiyo shizzle my nizzle crocodizzle 
							uhuh... yih! dapibizzle my shizz tempizzle sizzle. 
							Mauris pellentesque nibh et turpizzle. Vestibulum boom shackalack tortor.
							</p>
							<p>I'm in the shizzle izzle rhoncizzle nisi. My shizz pizzle habitasse platea dictumst. 
							Hizzle dapibus.  Phasellizzle own yo' volutpat tellus. Ut semper adipiscing lorem. 
							Donizzle shut the shizzle up est. Nulla sapien rizzle, ultrices nizzle, accumsizzle 
							vel, fermentizzle owned nizzle libero. 
							</p>
						</div>
					</div>
					<div class="moreposts right">
						<h3>More from the  blog</h3>
						<div class="minipost holder">
							<div class="left"><img src="http://placehold.it/33x33&text=blog" alt="blog"></div>
							<div class="left">
								<h4>Website Nonesense Drek</h4>
								<p>April 3rd, 2011 | 2 Comments</p>
							</div>
						</div>
						<div class="minipost holder">
							<div class="left"><img src="http://placehold.it/33x33&text=blog" alt="blog"></div>
							<div class="left">
								<h4>Cras rhoncus dapibus</h4>
								<p>March 20th, 2011 | 5 Comments</p>
							</div>
						</div>
						<div class="minipost holder">
							<div class="left"><img src="http://placehold.it/33x33&text=blog" alt="blog"></div>
							<div class="left">
								<h4>Habitasse lectus sit dapibus</h4>
								<p>February 15th, 2011 | 8 Comments</p>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="portfolio inner holder">
				<div class="rubric1 holder">
					<h2 class="left">New Portfolio Items</h2>
					<div class="line2 right"></div>
				</div>
			</div>
			<div class="widgets">
				<div class="inner holder">
				</div>
			</div>
			<div class="footer">
				<div class="inner holder">
				</div>
			</div>
		</div>
	</body>
</html>