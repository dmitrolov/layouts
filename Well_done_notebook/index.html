<! doctype html>
<html>
	<head>
		<meta charset="utf-8">
			<title>Note Book</title>
			<link rel="stylesheet" href="stylenote.css">
	</head>
	<body>
		<div class='pencil'>
		
		
		<div class='menutop'>
		</div>
			<div class='notebook'>
				<div class='note-top'>
					<h1>Heading 1</h1>
				</div>
				<div class='note-center'>
					<h2>Heading 2</h2>
					<h3>Heading 2</h3>
					<p>Lorem Ipsum - это текст - "рыба", часто используемый в  печати  и вэб-дизайне. 
					Lorem Ipsum является стандартной  "рыбой"   для  текстов   на   латинице   с  
					начала  XVI  века.  В  то  время  некий безымянный печатник  создал большую 
					коллекцию размеров и форм шрифтов, используя Lorem  Ipsum для  распечатки образцов. 
					Lorem Ipsum не только успешно пережил  без  заметных изменений пять веков, но и 
					перешагнул  в  электронный  дизайн.  Его  популяризации  в  новое  время   послужили 
					публикация  листов  Letraset  с образцами   Lorem  Ipsum  в  60-х  годах  и,  в 
					недавнее время, программы электронной вёрстки типа Aldus PageMaker, в шаблонах 
					которых используется Lorem Ipsum.</p>
					<p>Давно  выяснено, что  при  оценке  дизайна  и  композиции читаемый  текст  мешает 
					сосредоточиться. Lorem Ipsum используют потому, что тот обеспечивает более  или менее 
					стандартное заполнение шаблона, а  также   реальное  распределение   букв  и  пробелов  
					в  абзацах,  которое   не  получается  при  простой дубликации   "Здесь  ваш  текст..  
					Здесь  ваш  текст..  Здесь  ваш  текст.."  Многие  программы электронной вёрстки  и  
					редакторы  HTML  используют  Lorem Ipsum в качестве текста по умолчанию, так что  поиск  
					по  ключевым  словам  "lorem  ipsum"  сразу  показывает,  как  много веб-страниц всё ещё 
					дожидаются  своего  настоящего  рождения.  За  прошедшие  годы текст Lorem Ipsum получил 
					много версий. Некоторые  версии появились по ошибке, некоторые - намеренно (например, 
					юмористические варианты).</p>
				</div>
				<div class='note-bottom'></div>
			</div>
		</div>
		<div class='pozitions'>
			<div class='v1'>
				<div class='vizit'></div>
				<div class='logo'></div>
			</div>
			<div class='menuright'>
				<div class='blockmenu'>
					<a href='#' class="a1"></a>
					<a href='#' class="a2"></a>
					<a href='#' class="a3"></a>
					<a href='#' class="a4"></a>
				</div>
			</div>
			<div class='menuleft'>
				<div class='block-menu-left'>
					<a href='#' class="b1"></a>
					<a href='#' class="b2"></a>
					<a href='#' class="b3"></a>
					<a href='#' class="b4"></a>
					<a href='#' class="b5"></a>
					<a href='#' class="b6"></a>
					<a href='#' class="b7"></a>
					<a href='#' class="b8"></a>
				</div>
			</div>
			<div class='pen'></div>
		</div>
	</body>
</html>