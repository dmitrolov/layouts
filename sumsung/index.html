<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Hightlights</title>
    <link href="style.css" rel="stylesheet">
    <link href="fonts.css" rel="stylesheet">
</head>

<body>
    <div class="header">
        <div class="header_top">
            <div class="content-wrapp ">
                <div class="header_top-left">Galaxy Note20 | Note20 Ultra</div>
                <a href="https://samsungshop.com.ua/smartphones/Samsung-Galaxy-Note-20-Ultra-5G-N986F-Black" class="header_top_btn">
                    Купити зараз
                </a>
            </div>
        </div>
        <div class="header_bottom">
            <div class="content-wrapp">
                <ul class="menu">
                    <li class="active"><a href="main.html">ОСОБЛИВОСТІ</a>  </li>
                    <li ><a href="perfomance.html">ПРОДУКТИВНІСТЬ</a> </li>
                    <!-- <li><a href="">S Pen</a> </li>
                    <li><a href="">КАМЕРА</a> </li> -->
                    <li><a href="models.html">МОДЕЛІ</a></li>
                    <li><a href="accesories.html">АКСЕСУАРИ</a></li>
                </ul>
            </div>
        </div>
    </div>
    <section class="section top-block">
        <div class="top-block_wrapper">
            <img src="images/work.png" alt="">
            <h2>Комп'ютер у вашій кишені</h2>
        </div>

    </section>
    <section class="section pen">
        <div class="content-wrapp">
            <div class="text-info">
                <div class="text-info_wrpper">
                    <span class="text-info_subtitle">ВДОСКОНАЛЕНИЙ S PEN</span>
                    <h4 class="text-info_title">Відчуття справжньої ручки. Сила в багатьох рисах.</h4>
                    <p class="text-info_main">S Pen пропонує революційно низьку затримку, створюючи напрочуд реалістичне відчуття користування ручкою. У поєднанні з гнучкістю Bluetooth, керуванням жестами і ще багато іншим, це найпотужніше перо в сім’ї Galaxy.</p>
                </div>
            </div>
            <div class="pen-img">
                <!-- <img src="images/PEN.png" alt=""> -->
                <img src="images/PEN_with-shadow.png" alt="">
            </div>
        </div>
    </section>
    <section class="section notes">
        <div class="content-wrapp">
            <h3 class="section-title">
                Із неперевершеною легкістю перетворюйте свої нотатки на презентації PowerPoint
            </h3>
            <div class="main-info">
                <div class="main-info_item">
                    <span class="main-info_subtitle">НОТАТКИ В SAMSUNG NOTES</span>
                    <h4 class="main-info_title">Від письма поспіхом до охайного тексту одним дотиком</h4>
                </div>
                <div class="main-info_item">
                    <p class="main-info_text">За дотиком до вашого S Pen написане саме скоригується, якщо ви пишете з нахилом, забезпечуючи розбірливість та організованість тексту. Також можливе перетворення письма на текст, придатний для копіювання та вставляння. Після чого його можна експортувати як файл PPT або Word для легкого поширення.</p>
                </div>
            </div>
            <div class="note-example">
                <img src="images/notes.png" alt="">
            </div>
        </div>
    </section>
    <section class="section gaming">
        <div class="content-wrapp">
            <div class="gaming-img">
                <img src="images/gaming.png" alt="">
            </div>
            <div class="gaming-info">
                <div class="gaming-info_item">
                    <span class="gaming-info_subtitle">ІГРИ</span>
                    <h3 class="gaming-info_title">Перетворіть свій телефон на досконалу ігрову платформу</h3>
                </div>
                <div class="gaming-info_item">
                    <div class="gaming-info_main">Оновлений інтелектуальний процесор, удосконалена система охолодження, швидкий дисплей 120 Гц — усе це переводить гру на Galaxy Note20 і Note20 Ultra на цілком новий рівень. А при з’єднанні з мережею Wi-Fi вбудований засіб Wi-Fi Optimizer забезпечує швидкий зв’язок із низькими затримками, щоб гра йшла в режимі реального часу.</div>
                </div>
            </div>
        </div>
    </section>
    <section class="section connection">
        <div class="content-wrapp">
            <div class="tab-wrapper active">
                <div class="tab ">
                    <h3 class="section-title">
                        Надшвидкий 5G та Wi-Fi Optimizer переведуть вашу гру на новий рівень
                    </h3>
                    <div class="section-info">
                        <div class="section-info_item">
                            <span class="section-info_subtitle">Wi-Fi</span>
                            <h3 class="section-info_title">Швидке з’єднання і продуктивність до пари</h3>
                        </div>
                        <div class="section-info_item">
                            <div class="section-info_main">Потужний інтелектуальний процесор усередині Galaxy Note20 і Note20 Ultra гарантуватиме, що ваші завантаження і трансляції швидкі. А якщо поряд є точка доступу, Wi-Fi 6 забезпечить швидке й безпечне з’єднання, а Wi-Fi Optimizer — низькі затримки для безперервної гри і відеовикликів.</div>
                        </div>
                    </div>
                    <div class="connection_img">
                        <img src="images/connection-wifi.png" alt="">
                    </div>
                </div>
            </div>

            <div class="tab-wrapper hidden">
                <div class="tab ">
                    <h3 class="section-title">
                        Надшвидкий 5G та Wi-Fi Optimizer переведуть вашу гру на новий рівень
                    </h3>
                    <div class="section-info">
                        <div class="section-info_item">
                            <span class="section-info_subtitle">5G ТА WI-FI</span>
                            <h3 class="section-info_title">Надшвидке передавання даних і продуктивність йому пасує</h3>
                        </div>
                        <div class="section-info_item">
                            <div class="section-info_main">Потужний інтелектуальний процесор усередині Galaxy Note20 і Note20 Ultra забезпечить таку швидкість ваших завантажень і трансляцій, яка тільки можлива з 5G.А як поряд є точка доступу, Wi-Fi 6 забезпечить швидке й безпечне з’єднання, а Wi-Fi Optimizer — низькі затримки для безперервної гри і відеовикликів навіть у людних місцях.</div>
                        </div>
                    </div>
                    <div class="connection_img">
                        <img src="images/connection-5g.png" alt="">
                    </div>
                </div>
            </div>

        </div>
    </section>
    <section class="section camera">
        <div class="content-wrapp">
            <h3 class="section-title">
                Усі найкращі інструменти професійного фотографа, відеооператора, редактора
            </h3>
            <div class="camera-img_wrapper">
                <img src="images/camera.png" alt="">
            </div>
            <div class="camera_characteristics">
                <div class="camera_characteristics-list">
                    <p class="camera_characteristics-model">Galaxy Note20</p>
                    <ul>
                        <li><span class="value">3x</span>Гібридний оптичний зум</li>
                        <li><span class="value">30x</span>Зум із суперчіткістю</li>
                    </ul>
                </div>
                <div class="camera_characteristics-list">
                    <p class="camera_characteristics-model">Galaxy Note20 Ultra</p>
                    <ul>
                        <li><span class="value">5x</span>Оптичний зумZ</li>
                        <li><span class="value">50x</span>Зум із суперчіткістю</li>
                    </ul>
                </div>
            </div>
        </div>
    </section>
    <section class="section video">
        <div class="content-wrapp">
            <div class="section-info">
                <div class="section-info_item">
                    <span class="section-info_subtitle">ВІДЕО ПРОФЕСІЙНОГО РІВНЯ</span>
                    <h3 class="section-info_title">Досі найбільша роздільність запису відео, доступна на смартфоні.</h3>
                </div>
                <div class="section-info_item">
                    <div class="section-info_main">Відео 8K дає вам чітку картинку з приголомшливою деталізацією, а професійні засоби керування посилять ваші творчі можливості, даючи змогу редагувати прямо в камері. Зніміть суперплавний рух із частотою 120 кадрів/сек, виокремлюйте голоси за допомогою Bluetooth-мікрофона. А тоді додайте фінішні штрихи, відредагувавши відео прямо на своєму телефоні. </div>
                </div>
            </div>
        </div>
        <div class="video-img_wrapper">
            <img src="images/video.jpg" alt="">
        </div>
        <div class="headphones">
            <h5 class="headphones-title">Galaxy BEAN</h5>
            <p>Виступає Bluetooth-мікрофоном для професійного керування аудіо.</p>
            <a href="http://">Перейти до <span>Galaxy Buds Live</span></a>
        </div>
    </section>
    <section class="section samsung-share">
        <div class="samsung-share_monitor">
            <img src="images/samsung-share.png" alt="">
        </div>
        <div class="content-wrapp">

            <div class="section-info">
                <div class="section-info_item">
                    <span class="section-info_subtitle">Samsung DeX</span>
                    <h3 class="section-info_title">Дивіться тут, грайте там —  <br>усе й одразу</h3>
                </div>
                <div class="section-info_item hand">
                    <img src="images/samsung-share_hand.png" alt="">
                </div>
                <div class="section-info_item">
                    <div class="section-info_main">Samsung DeX було оновлено для бездротового з’єднання з другим екраном, що дасть вам більше можливостей завдяки роботі на більшому екрані. З’єднайтеся з монітором чи телевізором, щоб дивитися відео та відкривати галерею з телефона через спеціальний інтерфейс DeX UI і все ще мати змогу користуватися телефоном.</div>
                </div>
            </div>

        </div>

    </section>
    <section class="section battery">
        <div class="content-wrapp">
            <div class="section-info_item">
                <span class="section-info_subtitle">РОЗУМНИЙ АКУМУЛЯТОР НА ВЕСЬ ДЕНЬ</span>
                <h3 class="section-info_title">Повного заряду зранку вистачить до глибокої ночі</h3>
            </div>
            <div class="section-info_item">
                <div class="section-info_main">Galaxy Note20 і Note20 Ultra мають інтелектуальний акумулятор ємністю 4300 і 4500 мА·год (типове значення) відповідно з таким ефективним використанням заряду, що протримаються весь робочий день, ще й на нічні розваги вистачить. Відтепер у ваших руках не лише передові технології, а ще й енергія для тривалого користування ними. </div>
            </div>
        </div>
        <div class="battery-img">
            <img src="images/battery.png" alt="">
        </div>

    </section>
    <section class="section proximity">

        <div class="content-wrapp">

            <div class="section-info">
                <div class="section-info_item">
                    <span class="section-info_subtitle">ПРОСТЕ ПОШИРЕННЯ</span>
                    <h3 class="section-info_title">Сила близькості</h3>
                </div>
                <div class="section-info_item">
                    <div class="section-info_main">Удосконалена технологія просторової обізнаності забезпечує підвищену точність і більш інтуїтивну взаємодію, даючи змогу обмінюватися фото й відео з друзями простіше та швидше. Функція «Поділитися» (Point to Share) дає змогу просто навести телефон на телефон друга для передавання файлів.</div>
                </div>
            </div>
            <div class="proximity_img-wrapper">
                <img src="images/proximity.png" alt="">
            </div>
        </div>

    </section>
    <section class="section accessories">
        <div class="content-wrapp">
            <div class="section-info">
                <div class="section-info_item">
                    <span class="section-info_subtitle">АКСЕСУАРИ</span>
                    <h3 class="section-info_title">Більше способів зробити роботу й дозвілля продуктивнішими</h3>
                </div>
            </div>
        </div>
        <div class="accessories_img-wrapper">
            <img src="images/accessories.png" alt="">
        </div>
    </section>
    <section class="section banner">
        <div class="banner_bg">
            <div class="banner-wrapper">
                <div class="banner-icon"><img src="images/banner.svg" alt=""></div>
                <h3 class="banner-title">Дослідити Galaxy Note20 і Note20 Ultra в AR</h3>
                <button class="banner_btn">ДИВИТИСЯ</button>
            </div>
            <div class="banner">
                <img src="images/banner-hand.png" alt="">
            </div>
        </div>
    </section>

    <!-- <section class="section experience">
        <div class="content-wrapp">
            <h3 class="section-title">
                A powerful experience enhanced<br>
                with Galaxy favorites
            </h3>
            <ul class="experience-list">
                <li class="experience-list_item">
                    <div class="experience-list_item-icon">
                        <img src="images/OneUI.png" alt="">
                    </div>
                    <h4 class="experience-list_item-title">One UI 2.5</h4>
                    <h5 class="experience-list_item-subtitle">makes every movement intuitive</h5>
                    <p class="experience-list_item-text">Simplified and designed for clarity, the Galaxy experience feels natural with simple interactions that make sense.</p>
                </li>
                <li class="experience-list_item">
                    <div class="experience-list_item-icon">
                        <img src="images/Samsung-Knox.png" alt="">
                    </div>
                    <h4 class="experience-list_item-title">Samsung Knox</h4>
                    <h5 class="experience-list_item-subtitle">built-in security</h5>
                    <p class="experience-list_item-text">Samsung Knox offers end to end security in hardware and software to protect your data in real time.</p>
                </li>
                <li class="experience-list_item">
                    <div class="experience-list_item-icon">
                        <img src="images/authantication.png" alt="">
                    </div>
                    <h4 class="experience-list_item-title">Authentication</h4>
                    <h5 class="experience-list_item-subtitle">biometric protection</h5>
                    <p class="experience-list_item-text">Unlock your phone with Face Recognition or Ultrasonic Fingerprint. You can also choose a PIN, pattern, or password to secure your data.</p>
                </li>
                <li class="experience-list_item">
                    <div class="experience-list_item-icon">
                        <img src="images/ip68.svg" alt="">
                    </div>
                    <h4 class="experience-list_item-title">IP68</h4>
                    <h5 class="experience-list_item-subtitle">water and dust resistance</h5>
                    <p class="experience-list_item-text">Keeps your phone safe from splashes, drips, and up to 1.5 meters of water for around 
                        30 minutes.23</p>
                </li>
            </ul>

        </div>
    </section> -->



    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
    <script src="script.js" async></script>
</body>

</html>