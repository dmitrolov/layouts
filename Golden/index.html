<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt"logo" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles_first">
					Welcome To Our Studio!
				</div>
				<h1>It’s nice to Meet you</h1>
			</div>
			<a class="button" href="#">
				Tell me more
			</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="services clearfix">
				<div class="services__item">
					<img src="img/icon1.png" alt"Услуга">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="services__item">
					<img src="img/icon2.png" alt"Услуга">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="services__item">
					<img src="img/icon3.png" alt"Услуга">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt"Работа">
				<img src="img/img2.jpg" alt"Работа">
				<img src="img/img3.jpg" alt"Работа">
			</div>
		</div>
	</section>
	<section>
		<div class="container">
			<div class="title">
				<h2>About Us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="title_block clearfix">
				<div class="title_blockimg clearfix">
					<div class="title_text">
						<h4>July 2010</h4>
						<h5>Our Humble Beginnings</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.
						</p>
					</div>
					<img src="img/about1.jpg" alt"Про нас">
				</div>
				<div class="title_blockimg2 clearfix">
					<img src="img/about2.jpg" alt"Про нас">
					<div class="title_text2">
						<h4>January 2011</h4>
						<h5>Facing Startup Battles</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>
				<div class="title_blockimg clearfix">
					<div class="title_text">
						<h4>December 2012</h4>
						<h5>Enter The Dark Days</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.
						</p>
					</div>
					<img src="img/about3.jpg" alt"Про нас">
				</div>
				<div class="title_blockimg2 clearfix">
					<img src="img/about4.jpg" alt"Про нас">
					<div class="title_text2">
						<h4>february 2014</h4>
						<h5>Our Triumph</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="footer_text">
			<p>© Copyright 2014 FreebiesXpress.com</p>
		</div>
	</footer>
</body>
</html>